modules/nf-core/gemma/kinshipmatrix/test/main.nf.test

```diff
@@ -0,0 +1,76 @@
+nextflow_process {
+
+    name "Test Process GEMMA_KINSHIPMATRIX"
+    script "../main.nf"
+    process "GEMMA_KINSHIPMATRIX"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "gemma"
+    tag "gemma/kinshipmatrix"
+
+    // TODO nf-core: Change the test name preferably indicating the test-data and file-format used
+    test("gwas - vcf - pheno") {
+
+        when {
+            process {
+                """
+                // TODO nf-core: define inputs of the process here. Example:
+
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
```

Comment:
You do not use the `single_end` information in the process. So it isn't needed in the test
```suggestion
                    [ id:'test' ],
```